akka-javasdk/src/main/java/akka/javasdk/agent/ModelProvider.java

```diff
@@ -861,4 +862,348 @@ non-sealed interface Custom extends ModelProvider {
      */
     Object createStreamingChatModel();
   }
+
+  /** Settings for the Bedrock Large Language Model provider. */
+  static Bedrock bedrock() {
+    return new Bedrock(
+        "",
+        "",
+        false,
+        false,
+        false,
+        false,
+        -1,
+        -1,
+        Map.of(),
+        "",
+        Double.NaN,
+        Double.NaN,
+        -1,
+        Duration.ofMinutes(1),
+        2);
+  }
+
+  record Bedrock(
+      String region,
+      String modelId,
+      Boolean logRequests,
+      Boolean logResponses,
+      Boolean returnThinking,
+      Boolean sendThinking,
```

Comment:
is there a reason for using Boolean instead of boolean? is it for representing null? otherwise use primitive boolean